modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -641,6 +644,15 @@ public void setUp() throws Exception {
 				).name(
 					_OBJECT_FIELD_NAME_TEXT
 				).build(),
+				new TextObjectFieldBuilder(
```

Comment:
Why did you need to create a new field? This object already has a text object field. Also, I saw that there are already other object definitions with more than one text object field (`_objectDefinition1`, `_objectDefinition2` and `_objectDefinition3`)

Reply:
I was using **_objectDefinition5** because it already had a object entry created, but I don't need to use this existent entry more. And I needed to create a new field because this existent one is localized, what was impacting my expected flow in the test assertion. I updated the test to use **_objectDefinition1** instead and removed the extra field.